bus/bus.go

```diff
@@ -854,13 +854,23 @@ func (b *Bus) refreshContractV2(ctx context.Context, cs consensus.State, h api.H
 	}
 	prices := settings.Prices
 
+	// raise the renterFunds if they are too low to reach minNewCollateral
```

Comment:
I guess this is fine but it's not ideal because we pass in the `renterFunds` from the contractor and then use that value in our logging. I noticed that when reviewing the other PR. Quite annoying but wanted to bring it to your attention nonetheless

Reply:
It is quite annoying indeed. It's just an annoying chicken and egg problem with a lot of requirements to meet. e.g. more than the minimum allowance but less than max collateral.

The proper cleanup would be to change this endpoint but probably not worth it before the hardfork.